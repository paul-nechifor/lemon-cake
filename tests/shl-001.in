(<< 1 1)
(<< 1 2)
(<< 1 3)
(<< 0xf000000000000000 1)
(<< 1 63)
