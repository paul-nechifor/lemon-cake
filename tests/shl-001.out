2
4
8
-2305843009213693952
-9223372036854775808
